### PR TITLE
Fix nullable listener and injecting js in the correct page

### PR DESCRIPTION
**Pull request by fmicoInfo**

### CODE REVIEW

**Comment by victorg1991** [comment]
First PR 🎉 congrats!!

**Comment by jmnavarro** [comment]
👏🏻👏🏻👏🏻 well done @fmicoInfo!

File: android/library/src/main/java/com/liferay/mobile/screens/viewsets/defaultviews/portlet/PortletDisplayView.java [hunk 1]
```diff
@@ -92,7 +92,7 @@ public void showFinishOperation(String url, String body, final String injectedJs
 				@Override
 				public void onPageFinished(WebView view, String url) {
 					super.onPageFinished(view, url);
-					if (!isLoaded) {
+					if (!isLoaded && !url.contains("/c/portal/login")) {
```
Comment (nhpatt) [inline android/library/src/main/java/com/liferay/mobile/screens/viewsets/defaultviews/portlet/PortletDisplayView.java]: Can you extract this to a constant?

**Comment by dgarciasarai** [comment]
🎉 👏👏👏

File: android/library/src/main/java/com/liferay/mobile/screens/viewsets/defaultviews/portlet/PortletDisplayView.java (hunk 1 above)
Comment (fmicoInfo) [inline android/library/src/main/java/com/liferay/mobile/screens/viewsets/defaultviews/portlet/PortletDisplayView.java]: 👍

**Pull request merged**